getting-started/README.md

```diff
@@ -86,17 +86,17 @@ const provider = new NodeTracerProvider({
 provider.register();
 ```
 
-If you run your application now with `node -r ./tracing.js app.js`, your application will create and propagate traces over HTTP. If an already instrumented service that supports [Trace Context](https://www.w3.org/TR/trace-context/) headers calls your application using HTTP, and you call another application using HTTP, the Trace Context headers will be correctly propagated.
+Now, if you run your application with `node -r ./tracing.js app.js`, your application will create and propagate traces over HTTP. If an already instrumented service that supports [Trace Context](https://www.w3.org/TR/trace-context/) headers calls your application using HTTP, and you call another application using HTTP, the Trace Context headers will be correctly propagated.
 
-If you wish to see a completed trace, however, there is one more step. You must register an exporter to send traces to a tracing backend.
+However, if you want to see a completed trace, you need to register an exporter to send traces to a tracing backend.
 
-#### Initialize and Register a Trace Exporter
+#### Initialize and register a trace exporter
```

Comment:
Let's keep Title Case for headers

Reply:
@naseemkullah I was kind of trying to follow Google's style guide for developer documentation: https://developers.google.com/style/capitalization#capitalization-in-titles-and-headings

"In document titles and headings, use sentence case. That is, capitalize only the first word in the title, the first word in a subheading after a colon, and any proper nouns or other terms that are always capitalized a certain way."

Thoughts?

Reply:
Touché.

Reply:
What google's styleguide says goes imo, thanks for pointing that out